src/app/session-lobby/session-lobby.component.ts

```diff
@@ -9,13 +11,22 @@ import { TeacherSessionService } from '../teacher-session.service';
 export class SessionLobbyComponent implements OnInit {
 
   sessionID: string;
+  roundData: FirebaseRound;
 
-  constructor(public teacherSessionService: TeacherSessionService) {
+  constructor(public teacherSessionService: TeacherSessionService, public teacherRoundService: TeacherRoundService) {
     this.sessionID = 'demo-session'; // Temporary until a way to get the session is implemented
+    this.roundData = {flowerSpeciesIds: ['asclepias_syriaca', 'coreopsis_palmata'],
+                          status: 'fine',
+                          running: false,
+                          currentTime: 0, };
```

Comment:
We might want to have a separate interface for how we store the round "prototype". We will want to be able to specify the flower species in the round using direct references to the objects and also don't need to include things like `running: false`. I would suggest a round preset type that looks something like this:
```ts
interface PresetRound {
  flowerSpecies: FlowerSpecies[];
  startTime: TimePeriod;
  endTime: TimePeriod;
  tickSpeed: number;
  bees: {
    species: BeeSpecies;
    chance: number;
  }
}
```

Reply:
Yes, but probably in a separate pull request.